### PR TITLE
MdeModulePkg/Library/BootMaintenanceManagerUiLib: Handle timeout stat…

### PR DESCRIPTION
**Pull request by miczyg1**
…ically

### CODE REVIEW

File: MdeModulePkg/Library/BootMaintenanceManagerUiLib/BootMaintenanceManager.h [hunk 1]
```diff
@@ -161,7 +161,7 @@ typedef enum _TYPE_OF_TERMINAL {
 #define COM1_DATA_RATE_VAR_OFFSET       VAR_OFFSET (COM1DataRate)
 #define COM1_STOP_BITS_VAR_OFFSET       VAR_OFFSET (COM1StopBits)
 #define COM1_PARITY_VAR_OFFSET          VAR_OFFSET (COM1Parity)
-#define COM1_TERMINAL_VAR_OFFSET        VAR_OFFSET (COM2TerminalType)
+#define COM1_TERMINAL_VAR_OFFSET        VAR_OFFSET (COM1TerminalType)
```
Comment (mkopec) [inline MdeModulePkg/Library/BootMaintenanceManagerUiLib/BootMaintenanceManager.h:164]: Is this change supposed to go in this PR?

Comment (miczyg1) [reply MdeModulePkg/Library/BootMaintenanceManagerUiLib/BootMaintenanceManager.h:164]: Ahh I found this and decided to fix immediately... Anyway we don't use this at all... Up to you if you want me to separate it or not.

Comment (mkopec) [reply MdeModulePkg/Library/BootMaintenanceManagerUiLib/BootMaintenanceManager.h:164]: I'm okay with it, just wanted to make sure

**Review by mkopec — APPROVED** [review]

**Comment by mkopec** [comment]
Let's wait for CI to complete

**Pull request merged**